pg/seed2.pgsql

```diff
@@ -0,0 +1,6052 @@
+Illicium verum	Ba Jiao Hui Xian, Staranise tree	5	3	7-10	\N	LM	AN	SN	M	\N	\N	Y	12-Jan	5-Mar	10	H	\N	3	3
+Impatiens capensis	Jewelweed	1.2	0	\N	F	LMH	ANB	SN	M	\N	\N	\N	\N	10-Jul	\N	H	Bees, self	3	2
+Impatiens noli-tangere	Touch-Me-Not	1	0	5-9	\N	LMH	ANB	SN	M	\N	\N	\N	\N	9-Jul	10-Aug	H	Insects	3	2
+Incarvillea sinensis	\N	0.9	0	4-8	\N	LM	ANB	SN	M	W	\N	\N	\N	\N	\N	H	\N	1	1
+Indigofera cassioides	\N	3.5	0	8-11	\N	LM	ANB	N	M	\N	\N	Y	\N	\N	\N	H	Insects	1	1
+Indigofera heterantha	Indigo Bush	3	2.5	6-9	\N	LM	ANB	N	DM	\N	\N	N	11-May	9-Aug	\N	H	Insects	1	0
+Inga vera	Ice cream bean, River koko, Guaba, Pois Doux	15	15	10-12	F	MH	ANB	N	M	\N	\N	\N	\N	\N	\N	\N	Bees	2	2
+Iris cristata	Crested Iris, Dwarf crested iris	0.2	0.5	5-9	M	LM	AN	SN	M	\N	\N	\N	\N	6-May	\N	H	Insects	1	1
+Iris missouriensis	Rocky Mountain Iris	0.8	1	0-0	\N	LM	ANB	SN	MWe	\N	\N	\N	\N	6-May	\N	H	Insects	1	2
+Isodon japonicus	\N	1	0	\N	\N	LMH	ANB	SN	M	\N	\N	\N	\N	\N	\N	H	\N	1	1
+Jasminum grandiflorum	Jasmin	2.5	0.5	10-12	S	LMH	ANB	SN	M	\N	\N	\N	\N	\N	\N	\N	\N	3	3
+Jasminum humile	Yellow Jasmine	3	3	7-10	\N	LMH	ANB	SN	DM	\N	\N	\N	12-Jan	8-Jul	\N	H	Insects	1	1
+Juglans ailanthifolia cordiformis	Heartseed Walnut	20	15	4-8	M	LMH	ANB	N	M	\N	\N	N	\N	6	\N	M	Wind	4	1
+Juglans cinerea	Butternut - White Walnut, Butternut	25	20	3-7	M	LMH	ANB	N	M	\N	\N	N	\N	6-May	11-Oct	M	Wind	3	3
+Juglans mandschurica	Manchurian Walnut	20	0	4-8	\N	LMH	ANB	N	M	\N	\N	\N	\N	6-May	10-Sep	M	Wind	3	1
+Juglans nigra	Black Walnut	30	20	4-9	F	LMH	ANB	N	M	\N	\N	N	\N	6-May	10	M	Wind	3	3
+Juglans regia kamaonia	Walnut	30	0	4-8	\N	LMH	ANB	N	M	\N	\N	N	\N	6-May	\N	M	Wind	3	3
+Juglans x bisbyi	Buartnut	20	0	4-8	\N	LMH	ANB	N	M	\N	\N	\N	\N	6-May	\N	M	Wind	3	0
+Kalimeris pinnatifida	Japanese Aster	1.5	0	5-8	M	LMH	ANB	SN	M	\N	\N	\N	\N	10-Aug	\N	H	Insects	1	0
+Kerria japonica	Bachelor's Button, Japanese rose, Jew's Mallow, Japanese Kerria	2	2	4-9	M	LMH	ANB	SN	M	\N	\N	\N	\N	5-Apr	\N	M	Insects	1	1
+Knightia excelsa	Rewarewa	20	4	8-11	\N	LMH	ANB	SN	M	\N	\N	Y	12-Jan	8-Jul	\N	H	Bees	1	0
+Lactuca pulchella	Blue Lettuce	1	0	\N	\N	LM	ANB	SN	M	\N	\N	\N	\N	\N	\N	H	Insects	2	2
+Lactuca sativa capitata	Cabbage Lettuce	0.9	0.3	5-9	\N	LM	NB	SN	M	\N	\N	N	\N	8-Jul	9-Aug	H	Flies, self	3	3
+Lagenaria siceraria	Bottle Gourd	9	0	10-12	F	LMH	ANB	N	M	\N	\N	Y	\N	9-Aug	\N	M	Insects	3	2
+Lagoecia cuminoides	Wild Cumin	0.3	0	\N	\N	LMH	ANB	SN	M	\N	\N	\N	\N	\N	\N	H	\N	1	0
+Lansium domesticum	Langsat, Lansa, Lanzon	15	12	11-12	S	MH	ANB	SN	M	\N	\N	Y	\N	\N	\N	\N	Insects	4	2
+Laportea canadensis	Canadian Wood Nettle	1	1	0-0	\N	LMH	ANB	SN	M	\N	\N	\N	\N	8-May	\N	\N	\N	3	1
+Larix occidentalis	Western Larch	45	0	2-6	F	LM	ANB	N	M	W	\N	N	\N	\N	10	M	Wind	2	2
+Lathyrus odoratus	Sweet Pea, Wild Pea,Vetchling	2	0	2-11	F	LMH	ANB	SN	M	\N	\N	\N	\N	9-Jul	10-Aug	H	Insects	1	0
+Lathyrus palustris	Slenderstem Peavine, Marsh pea	1.2	0	4-8	\N	LMH	ANB	N	MWe	\N	\N	\N	\N	8-Jul	\N	H	Insects	1	0
+Lavandula angustifolia	English Lavender, True Lavender	1.2	1	5-8	S	LMH	ANB	N	DM	M	\N	
```

Comment:
Sooo much data!!! haha but that is very cool and necessary since we need recommendation for all the zipcodes